_posts/2022-01-05-cockpit-260.md

```diff
@@ -0,0 +1,77 @@
+
+## Client: Support port specification
+ 
+Cockpit Client now supports specifying the port number when connecting, with the usual syntax like `user@host.example.net:22222`.  This is the same form used in Cockpit bastion host configurations.
+
+## Certificate login validation: Action required on updates
+
+Earlier Cockpit/sssd versions did not check trust or revocation status of a presented client certificate, and thus certificate/smart card login was secure and supported only when matching the entire binary certificate against the Identity Management's database. With sssd 2.6.1 and Cockpit 260, the certificate signature and revocation status is now validated against the CA [configured in sssd](https://www.mankier.com/5/sssd.conf#Services_Sections-PAM_configuration_options), and any non-trusted certificate is rejected. This includes the case when the local sssd has *no* configured CA, which may break certificate logins after updating cockpit and sssd.
+
+Thus if you use certificate login, you need to set up the trusted CA in sssd. Please see the [certificate authentication documentation](https://cockpit-project.org/guide/latest/cert-authentication.html#certauth-server-cockpitconf) for details.
+
+This issue was assigned [CVE-2021-3698](https://access.redhat.com/security/cve/CVE-2021-3698).
+
+## Client: Show previously used hosts
+
+Cockpit client now shows list of hosts that were used before so you can log in with one click. It is also possible to remove unwanted hosts from this list.
+
+![Screenshot from 2021-12-10 08-09-06]({{ site.baseurl }}/images/260-client-show-previously-used-hosts.png)
+
+
+## Create container in pod
+
+It is now possible to create a container in an existing pod using the `Create container in pod` button.
+
+![screenshot of ## create container in pod]({{ site.baseurl }}/images/260-create-container-in-pod.png)
+
+Create container dialog now shows in which pod the container is created
+
+![screenshot of ## create container in pod]({{ site.baseurl }}/images/260-create-container-in-pod-2.png)
+
+## Podman restart policy
+
+For system containers it is now possible to set a restart policy for new containers.
+
+![screenshot of ## podman restart policy]({{ site.baseurl }}/images/260-podman-restart-policy.png)
+
+## Allow inserting multiple environment variables
+
+Podman users can now insert multiple environment variables in bulk by copy pasting an list of environment files the variables have to be formatted as FOO=bar separated by newlines.
+
+
+## Try it out
+
+Cockpit 260 and cockpit-podman 39 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/260)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
```

Comment:
[260 cockpit release](https://bodhi.fedoraproject.org/updates/?packages=cockpit) done as well now, so URLs can be updated.